tests/serverpod_test_server/test_integration/test_tools/authentication_validation_test.dart

```diff
@@ -131,7 +131,7 @@ void main() {
             'and the authenticated user is revoked then stream is closed with ConnectionClosedException.',
             () async {
           await expectLater(
-            session.messages.authenticationRevoked(
+            session.serverpodSession.messages.authenticationRevoked(
```

Comment:
Question: Now that the "session" is not really returned, should we change the name of session in the `withServerpod` interface to reflect this?

Reply:
As discussed offline we'll create a follow-up PR for this 👍